typespec-tests/src/main/java/com/cadl/multipart/models/FormData.java

```diff
@@ -0,0 +1,154 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+// Code generated by Microsoft (R) TypeSpec Code Generator.
+
+package com.cadl.multipart.models;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.annotation.Generated;
+import com.azure.core.util.BinaryData;
+import com.azure.json.JsonReader;
+import com.azure.json.JsonSerializable;
+import com.azure.json.JsonToken;
+import com.azure.json.JsonWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * The FormData model.
+ */
+@Fluent
+public final class FormData implements JsonSerializable<FormData> {
+    /*
+     * The name property.
+     */
+    @Generated
+    private final String name;
+
+    /*
+     * The image property.
+     */
+    @Generated
+    private final BinaryData image;
+
+    /*
+     * The filename for image
+     */
+    @Generated
+    private String imageFilename = "image";
```

Comment:
this "filename" property is generated by codegen

Reply:
Where is default file name defined?

Reply:
Currently using the field name, aka `image` in this case.

If user does not provide the filename, current assumption is that the filename does not matter. So `image` or random UUID should both work (as long as there is no duplicate).